youtube_dl/extractor/iai.py

```diff
@@ -0,0 +1,53 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class IaiIE(InfoExtractor):
+    IE_NAME = 'IAI'
+    _VALID_URL = r'https://iai.tv/video/(?P<id>[^/]+)'
```

Comment:
Will capture incorrect id for https://iai.tv/video/darkness-authority-and-dreams#blah.